driver-core/src/main/com/mongodb/internal/operation/CreateIndexesOperation.java

```
@@ -284,6 +309,9 @@ private BsonDocument getCommand(final ConnectionDescription description) {
         command.put("indexes", new BsonArray(values));
         putIfNotZero(command, "maxTimeMS", maxTimeMS);
         appendWriteConcernToCommand(writeConcern, command, description);
+        if (commitQuorum != null && serverIsAtLeastVersionFourDotFour(description)) {
```

The specification requires that the driver throw an exception in this case instead of silently ignoring it.

---

Make sure you add a test for it as well.

---

A `MongoClientException` is now thrown and I have added a test for it.